business_register/converter/business_converter.py

```diff
@@ -4,10 +4,22 @@
 logger.setLevel(logging.INFO)
 
 from business_register.models.kved_models import Kved
+from business_register.models.company_models import Company
 from data_ocean.converter import Converter
 from data_ocean.models import Authority, Status, TaxpayerType
 
 
+def find_company_by_edrpou(edprou):
+    if edprou == '00000000':
+        return None
+    queryset = Company.objects.filter(edrpou=edprou)
+    if len(queryset) == 1:
+        return queryset[0]
+    if not len(queryset):
+        return None
+    return queryset.order_by('status').first()
```

Comment:
maybe like this? 6 > 1
```suggestion
    return Company.objects.filter(edrpou=edprou).order_by('status').first()
```